applications/ultrasound_segmentation/cpp/CMakeLists.txt

```diff
@@ -95,6 +95,7 @@ if(BUILD_TESTING)
     WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
   set_tests_properties(ultrasound_segmentation_cpp_test PROPERTIES
                        ENVIRONMENT "LD_LIBRARY_PATH=$ENV{LD_LIBRARY_PATH}:${CMAKE_BINARY_DIR}/gxf_extensions"
+                       TIMEOUT 900
```

Comment:
Is this different from the 1-hour TRT generation issue previously reported?

Reply:
This is the only test that fails currently on CAGX so probably unrelated